languages/ruby/exercises/concept/floating-point-numbers/.docs/after.md

```diff
@@ -0,0 +1,8 @@
+You can find a short introduction to floating-point numbers at [0.30000000000000004.com][0.30000000000000004.com].
+
+The [Float Toy page][evanw.github.io-float-toy] has a nice, graphical explanation how a floating-point numbers' bits are converted to an actual floating-point value.
+
+To repeatedly execute logic, one can use loops. In this example the `while` loop was useful because it keeps on looping until a boolean condition evaluates to `true`.
```

Comment:
```suggestion
To repeatedly execute logic, one can use loops. In this example the `while` loop is useful because it keeps on looping until a boolean condition evaluates to `true`.
```